cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/segment/FindSegmentSimilarPostHandler.java

```diff
@@ -2,6 +2,8 @@
 
 import io.javalin.http.Context;
 import io.javalin.plugin.openapi.annotations.*;
+import io.javalin.plugin.openapi.dsl.OpenApiBuilder;

```

Comment:
I think in this file the OpenAPI annotation for performPost() should also be removed?